data/mods/DinoMod/monstergroups/wilderness.json

```
@@ -12,6 +12,20 @@
         "pack_size": [ 4, 8 ],
         "conditions": [ "DUSK", "DAWN", "SPRING", "SUMMER", "AUTUMN" ]
       },
+      {
+        "monster": "mon_kosmoceratops",
+        "freq": 21,
+        "cost_multiplier": 25,
+        "pack_size": [ 4, 9 ],
+        "conditions": [ "DUSK", "DAWN", "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_amargasaurus",
+        "freq": 12,
+        "cost_multiplier": 25,
+        "pack_size": [ 4, 12 ],
+        "conditions": [ "DUSK", "DAWN", "SPRING", "SUMMER", "AUTUMN" ]
```

```suggestion
        "conditions": [ "NIGHT", "SPRING", "SUMMER", "AUTUMN" ]
```
It's a South American dino, but it's imaginable that it would eventually migrate north so fair enough. Adult sauropods are night only in Forests due to player complaints.